e2e/e2e_test.go

```diff
@@ -0,0 +1,77 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
```

Comment:
nit: can you please sort this imports?

Reply:
it's somehow sorted by my IDE every time I save the file